### ISSUE TITLE
Pinned tabs overflow when tab bar scrolling is activated

### ISSUE THREAD

**Issue by vctls** [BGMP/firefox-revert-proton#4]
Thanks for the style 😊
I found only one problem with it:

Pinned tabs usually look like this:
![image](https://user-images.githubusercontent.com/18054077/131998839-66deff9f-695f-48be-ac06-39f71a62d47e.png)

But if you open enough tabs to activate scrolling, the layout breaks:
![image](https://user-images.githubusercontent.com/18054077/131998828-3fb44f3b-1685-41be-9785-ebca084afff7.png)

**Comment by BGMP** [comment]
Kind of an old issue, but I think I found out why the first tab in the list was overflowing to the right (check references). I'm pretty sure that in original Photon there used to be some sort of separator or something between pinned and normal tabs... can't quite remember.

Would be cool if you could tell me if it breaks something @vctls !

**Comment by vctls** [comment]
Pinned tabs work, but I've got a few other issues:
![image](https://user-images.githubusercontent.com/18054077/202897610-cb882dd6-9aa5-4ebd-b993-c7a9162b551b.png)
The extension icons in the address bar worked until a recent FF update, so I don't think it has anything to do with your recent changes, but it would need a fix nonetheless.  
The dark color on the light theme, on the other hand, is new.

To be honest, I was about to drop these modifications and try to get used to the new look, because maintaining a big userChrome through each FF update is a bit tedious.
The only thing that really bothers me is how they highlight the active tab, and how containerized tabs stand out much more, but I guess I can get used to that as well…

**Comment by BGMP** [comment]
Nice to see you're still around, @vctls !

Seems like I got here just in time then! I'm planning on maintaining these styles indefinitely, mainly because I use Firefox and enjoy the looks of Photon.

I'm having a hard time replicating what's on your screenshot. What's your exact setup? Things such as your FF version, what extension icons you mean and how you're enabling light mode would be helpful things to know.

I've tried the latest userChrome on:
- Firefox v107.0
- Firefox Developer Edition v108.0b4

They don't break on my end, but I'm pretty sure I'm missing something from that screenshot.

Edit: Okay, switching themes in FF seems to break some things. I think it's breaking because I originally designed my fork to work on **Dark**, and I kind of force-merged everything in lol:

![Capture](https://user-images.githubusercontent.com/26081543/202955948-bf7c3eb6-25a7-400f-95d6-915fe1da0d01.PNG)

I will try to come up with some updates that will work with all the stock colour themes, but this doesn't explain why your icons are breaking. They all appear fine on my end. Let me know the theme you're using too.

**Comment by vctls** [comment]
I'm using FF Developer Edition 108 and automatic system theme (light/dark).  
I think the icons breaking in the address bar are from extensions. Namely, Xdebug Helper, Tab Stash, Form History Control and FF Multi-Account Containers.  
I'm using a couple modifications of my own too, mostly to change menu items.

**Comment by BGMP** [comment]
Hey @vctls,

Thanks for the extra info. I investigated and came up with a couple of changes which may help the issue. Try out compact or normal from [my branch](https://github.com/BGMP/firefox-revert-proton/tree/dev-bgm) and let  me know if it helps!

Thanks for the patience!

Edit: I did some extra testing myself and it seems to work fine. Let me know if something did break on your end though. I pushed everything to trunk now.
